specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-03-01/network.json

```diff
@@ -0,0 +1,168 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "NetworkManagementClient",
+    "description": "The Microsoft Azure Network management API provides a RESTful set of web services that interact with Microsoft Azure Networks service to manage your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Networks service.",
+    "version": "2018-02-01"
```

Comment:
Why the version is still `2018-02-01` but the folder is `2019-03-01`?

Reply:
This swagger only includes common model definitions used in other APIs, so the version is unused. I can update it for consistency.

---
In reply to: [285733800](https://github.com/Azure/azure-rest-api-specs/pull/5918#discussion_r285733800) [](ancestors = 285733800)